lib/pbench/server/indexer.py

```diff
@@ -135,7 +136,20 @@ def get_es(config, logger):
     # file instead of setting the logging level up so high.
     logging.getLogger("urllib3").setLevel(logging.FATAL)
     logging.getLogger("elasticsearch1").setLevel(logging.FATAL)
-    return Elasticsearch(hosts, max_retries=0)
+    ssl_context = create_default_context(cafile=config.get("Indexing", "cert_location"))
+    ssl_context.check_hostname = True
+    ssl_context.verify_mode = CERT_REQUIRED
+    elastic_user = config.get("Indexing", "elastic_user")
+    elastic_password = config.get("Indexing", "elastic_password")
+    es = Elasticsearch(
+        hosts,
+        use_ssl=True,
+        http_auth=(elastic_user, elastic_password),
+        ssl_context=ssl_context,
+        verify_certs=True,
+        max_retries=0,
+    )
+    return es
```

Comment:
This and `_bulk_dispatch` look very similar if not identical; it'd be a good idea to consolidate a single definition somewhere in `pbench.server` or `pbench.server.utils`.